### DISCUSSION TITLE
Warum werden USB-Devices bei Neustart oder Update neu formatiert?

### DISCUSSION THREAD

**Discussion by kumzugloom** [OpenCCU/OpenCCU#2196] [🇩🇪 Fragen&Antworten] [answered]
Wann immer ich meinen Homatic Raspi neu starte, ob durch Update oder einfach so ist danach der komplette USB-Speicher leer. 
Ist das ein Fehler, oder soll das so sein?
Auf diese Weise sind nun alle Diagrammdaten verschwunden und auch manuelle Datenpunkte, die ich per Skript aufgezeichnet und extra auf USB gespeichert hatte, damit sie auch Updates überleben. Die Diagramme sind nun sogar komplett kaputt:

> "Es ist kein USB-Speicher für die Speicherung der Diagrammdaten vorhanden oder auf diesen konnte nicht zugegriffen werden. Um die Diagrammdaten zu speichern, ist externer USB-Speicher erforderlich."

Während das Skript wieder fleißig Daten nach /media/usb1/ schreiben kann. Soll das so? Was mache ich falsch?

**Comment by jens-maus** [comment, marked as answer]
Das hört sich stark nach einem defekten USB Speicher an der ggf. beim Neustart einfach seine Daten verliert, etc. RaspberryMatic löscht definitiv nichts beim hochfahren oder legt den USB Stick neu an. Ich würde daher vorschlagen einfach einen komplett frischen USB-Stick mal zu verwenden und zu testen das die Daten nach einem Neustart über die Web-Oberfläche immer noch korrekt vorhanden sind (kein hartes Ausschalten des RaspberryPi!).

**Reply by kumzugloom** [reply]
Oh wow. Das war ja eine extrem schnelle Antwort!
Und erste Tests scheinen deine Vermutung zu bestätigen. Hätt' ich auch gleich selbst drauf kommen können, war aber offenbar so sicher, dass es funktioniert, weil die Daten ja gespeichert werden konnten.
Vielen Dank dir! :)
